test/integration/framework/util.go

```diff
@@ -19,9 +19,22 @@ limitations under the License.
 package framework
 
 import (
+	"io/ioutil"
+	"net/http/httptest"
 	"strings"
+	"testing"
+	"time"
 
+	"github.com/golang/glog"
+
+	"k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
 	clientset "k8s.io/client-go/kubernetes"
+	"k8s.io/kubernetes/pkg/api"
```

Comment:
this has been recently moved to `./pkg/apis/core` i hope that was the error.

Reply:
indeed (my PR, I should know :-) ). Must be rebased / adapted.

Reply:
LOL :)

Reply:
Done now in the other PR, will ping you all once the PR is ready for review.